### PR TITLE
Various improvements (→ to use peakina in toucantoco backend)

### PR DESCRIPTION
**Pull request by fspot**
- add xml mimetype support with xmltodict+jq
- expose `is_matching(filename)` method on datasource
- and some more misc commits

### CODE REVIEW

File: peakina/readers/xml.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Module to add xml support
+"""
+from typing import Union
+
+import pandas as pd
+import xmltodict
+from jq import jq
+
+
+def transform_with_jq(data: Union[dict, list], jq_filter: str) -> list:
```
Comment (virobidou) [inline peakina/readers/xml.py:11]: Can you also add a jq filter for json files ?

Comment (fspot) [reply peakina/readers/xml.py:11]: done :+1:

**Review by PrettyWood — CHANGES_REQUESTED** [review]
https://github.com/ToucanToco/peakina/pull/3/commits/de6853f8574c74f9253a62e16ae4ebd6a2d2c3f0 is empty @fspot ! Otherwise LGTM !

**Review by PrettyWood — APPROVED** [review]
:clap:

**Pull request merged**